services/minespace-web/src/styles/tools/syncfusion_imports.scss

```diff
@@ -1,17 +1,17 @@
-@import "~@syncfusion/ej2-base/styles/material.scss";
-@import "~@syncfusion/ej2-buttons/styles/material.scss";
-@import "~@syncfusion/ej2-dropdowns/styles/material.scss";
-@import "~@syncfusion/ej2-icons/styles/material.scss";
-@import "~@syncfusion/ej2-inputs/styles/material.scss";
-@import "~@syncfusion/ej2-navigations/styles/material.scss";
-@import "~@syncfusion/ej2-popups/styles/material.scss";
-@import "~@syncfusion/ej2-splitbuttons/styles/material.scss";
-@import "~@syncfusion/ej2-lists/styles/material.scss";
-@import "~@syncfusion/ej2-layouts/styles/material.scss";
-@import "~@syncfusion/ej2-grids/styles/material.scss";
-@import "~@syncfusion/ej2-react-pdfviewer/styles/material.scss";
-@import "~@syncfusion/ej2-react-filemanager/styles/material.scss";
+@import "~@syncfusion/ej2-base/styles/material.css";
```

Comment:
Interesting.  Was this a change on the Syncfusion side of things?  Or were there never scss versions of these?